src/main/resources/templates/login.html

```diff
@@ -33,25 +33,23 @@
                             <h3 class="dashhead loginpanel" th:text="#{login.header}">Log In</h3>
                         </div> <!-- panel-heading -->
                         <div class="welcome">
-                        <p> <b>Welcome to ncl.sg </b></p>
+                        <p><b>Welcome to ncl.sg </b></p>
                         </div>
                         <p>&nbsp;</p>
 
                         <div class="panel-body loginform">
-                            <!-- TODO hard code login details for now -->
                             <form method="post"  action="#" th:action="@{/login}" th:object="${loginForm}" role="form">
 	                            <fieldset>
-	                                <div class="form-group text-danger" th:if="${loginForm != null and loginForm.errorMsg != null}" th:text="${loginForm.errorMsg}">Invalid email/password.</div>
+	                                <div class="form-group account-change-fail" th:if="${loginForm != null and loginForm.errorMsg != null}">
+                                        <i class="fa fa-exclamation-circle"></i> <span th:text="${loginForm.errorMsg}"></span>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)